client/dashboard/task-list/tasks/steps/plugins.js

```diff
@@ -108,6 +108,16 @@ class Plugins extends Component {
 			return null;
 		}
 
+		if ( 0 === pluginSlugs.length ) {
```

Comment:
We can move this above:

```
		if ( autoInstall ) {
			return null;
		}
```

To avoid the `autoInstall={ ! extensionInstallError }` used earlier.